release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| 12. | **Feature** | Exposed application type provision timestamp | **Brief Description**: Added application type provision timestamp that is exposed via existing application type API's.<br>**Impact**: To better support Service Fabric Resource Provider (SFRP) and Service Fabric managed clusters (SFMC) scenarios, SF runtime needs to allow support for additional metadata for ARM deployed resources.<br>**Documentation Reference**: Read more about it at - <ul><li>ApplicationTypeMetadata</li></ul> |
+| 13. | **Feature** | Exposed API for reporting MoveCost | **Brief Description**: Exposed API for reporting MoveCost on behalf of other service replicas and partitions.<br>**Impact**: MoveCost could be changed with action outside of partition. If clients don't own service logic, they could use only static load.<br>**Solution**: New API allows reporting MoveCost from outside of the service, which could be beneficial for clients that don't have service logic.<br>**Documentation Reference**: Read more about it at - <ul><li>[Reporting move cost for a partition](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-resource-manager-movement-cost#reporting-move-cost-for-a-partition)</li></ul>**Workaround**: Replicas could report MoveCost values for themselves or clients could specify static MoveCost value during service creation. |
+| 14. | **Feature** | Create replacement replicas before Replica Restart Wait Duration (RRWD) expires when there are less than min replicas up | **Brief Description**: Starting with Service Fabric version 9.0, we have ability to adjust logic around Replica Restart Wait Duration.<br>Currently, when replica goes down, SF waits for a configurable time period until decides to build a new replica elsewhere, hoping the existing replica might become available again.<br>The problem here is that, when partition goes below MinReplicaSetSize number of replicas, it will still wait for that period of time before bringing new replica up, causing unavailability for the customer.<br>By toggling IgnoreReplicaRestartWaitDurationWhenBelowMinReplicaSetSize on, SF will not wait for problematic replica, but start building new one instantly, making sure that there is at least MinReplicaSetSize replicas available as soon as possible.<br>This configuration is dynamic, and customer can toggle the behavior accordingly.<br>The default value for this config is false, meaning that it is switched off.<br>**Impact**: SQL MI asked for the feature as a way to increase availability.<br> |
```

Comment:
SQL MI asked for the feature as a way to increase availability. -> Is this something that we would want to expose externally?